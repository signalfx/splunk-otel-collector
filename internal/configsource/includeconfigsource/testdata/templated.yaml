config_sources:
  include:
  include/watch:
    watch_files: true

component_0: |
  $include: ./testdata/component_template
  glob_pattern: /var/**/*.log
  format: json

component_1: $include:./testdata/no_params_template

component_2: |
  $include: ./testdata/yaml_template
  k0: 42
  cfg:
    k0: a_string
    k1: true

component_3:
  scalar_data: ${include/watch:./testdata/scalar_data_file}
  yaml_data: $include/watch:./testdata/yaml_data_file
