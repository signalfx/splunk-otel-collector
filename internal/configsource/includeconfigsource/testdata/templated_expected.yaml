component_0:
  logs_path: /var/**/*.log
  log_format: json

component_1:
  bool_field: true

component_2:
  int_field: 42
  map:
    str_field: a_string
    bool_field: true

component_3:
  scalar_data: 42
  yaml_data:
    field: value
    map:
      k0: 42
      k1: v1

